screens/Home.js

```diff
@@ -161,8 +161,8 @@ class HomeScreen extends React.Component {
             }
           >
             <View style={{ flex: 1, alignItems: "center" }}>
-              <Text style={theme.text}>
-                No workouts found, create a new one
+              <Text style={[theme.text, {textAlign: 'center'}]}>
+                Click + to create a new workout or find more in the Discover Tab
```

Comment:
It would be great to add some graphics to this screen as well like the one shown in [the article's "Add personality to your empty state" section](https://www.justinmind.com/blog/everything-you-need-to-know-about-empty-state-design/).

Reply:
In the future maybe